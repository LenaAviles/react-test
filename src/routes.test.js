import React from 'react';
import Routes from './routes';
import ShallowRenderer  from 'react-test-renderer/shallow';

it('renders Routes correctly', () => {
  const renderer = new ShallowRenderer()
  const tree = renderer
    .render(<Routes />);
  expect(tree).toMatchSnapshot();
});