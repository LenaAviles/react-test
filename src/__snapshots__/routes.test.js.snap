// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Routes correctly 1`] = `
<Layout>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/add-question"
    />
  </Switch>
</Layout>
`;
